reactor-core-micrometer/src/main/java/reactor/core/observability/micrometer/MicrometerObservationListener.java

```diff
@@ -98,7 +98,6 @@ final class MicrometerObservationListener<T> implements SignalListener<T> {
 		//NOTE: we don't use the `DocumentedObservation` features to create the Observation, even for the ANONYMOUS case,
 		//because the discovered tags could be more than the documented defaults
 		tapObservation = defaultObservation(configuration, observationSupplier)
```

Comment:
The name `defaultObservation` is misleading, it should be changed because of issues like this one, e.g. `getOrCreateObservation`.